uwblueprint/community-fridge-kw
# fix: add logic for creating schedules for recurring donations

## Notion ticket link

[Editing the create schedule endpoint](https://www.notion.so/uwblueprintexecs/Editing-the-create-schedule-endpoint-847a758ef1b04f8db226e0a79a2519fa)



## Implementation description
* add logic for creating schedules for recurring donations



## Steps to test
1. use Postman to create schedules with `frequency` daily, weekly, and monthly
2. verify in pgadmin if the correct number of objects were created with the correct fields



## What should reviewers focus on?
* correctness of values in database


## Checklist
